src/prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 generator client {
   provider        = "prisma-client-js"
-  binaryTargets   = ["debian-openssl-3.0.x", "linux-musl", "darwin", "native", "rhel-openssl-1.0.x"]
+  binaryTargets   = ["debian-openssl-1.1.x", "linux-musl", "darwin", "native", "rhel-openssl-1.0.x"]
```

Comment:
this should be 3

Reply:
Thanks! Not sure how this changed. Should explain the mysterious staging issue I had